File: solutions/0280. Wiggle Sort/001.java
class Solution {
    void swap(int [] nums, int i, int j) {
        int tmp = nums[i];
        nums[i] = nums[j];
        nums[j] = tmp;
    }
    void wiggleSort(int [] nums, int i) {
        if(i>=nums.length-1) return;
        if(nums[i] > nums[i+1]) {
            swap(nums, i, i+1);
        }
        i++;
        if(i>=nums.length-1) {
            return;
        };
        if(nums[i]< nums[i+1]) {
            swap(nums, i, i+1);
        }
        i++;
        wiggleSort(nums,i);
    }
    public void wiggleSort(int[] nums) {
        wiggleSort(nums,0);
    }
}

File: solutions/0217. Contains Duplicate/001.py
class Solution:
    # @param {integer[]} nums
    # @return {boolean}
    def containsDuplicate(self, nums):
        if not nums or len(nums) == 1:
            return False;
        nums_tmp = sorted(nums);
        for i in range(len(nums_tmp) - 1):
            if nums_tmp[i] == nums_tmp[i+1]:
                return True;
        return False;
            
        